sympy/functions/elementary/trigonometric.py

```diff
@@ -1568,6 +1568,97 @@ def taylor_term(n, x, *previous_terms):
                     bernoulli(2*k)*x**(2*k - 1)/factorial(2*k))
 
 
+class sinc(TrigonometricFunction):
+    r"""Represents unnormalized sinc function
+
+    Examples
+    ========
+
+    >>> from sympy import sinc, oo, jn, Product, Symbol
+    >>> from sympy.abc import x
+    >>> sinc(x)
+    sinc(x)
+
+    Automated Evaluation
+
+    >>> sinc(0)
+    1
+    >>> sinc(oo)
+    0
+
+    Differentiation
+
+    >>> sinc(x).diff()
+    (x*cos(x) - sin(x))/x**2
+
+    Series Expansion
+
+    >>> sinc(x).series()
+    1 - x**2/6 + x**4/120 + O(x**6)
+
+    As zero'th order spherical Bessel Function
+
+    >>> sinc(x).rewrite(jn)
+    jn(0, x)
+
+    Product form
+
+    >>> k = Symbol('k', integer=True, positive=True)
+    >>> sinc(x).rewrite(Product)
+    Product(2**(-k)*cos(x), (k, 1, oo))
+
+    References
+    ==========
+
+    .. [1] http://en.wikipedia.org/wiki/Sinc_function
+
+    """
+
+    def fdiff(self, argindex=1):
+        x = self.args[0]
+        if argindex == 1:
+            return (x*cos(x) - sin(x)) / x**2
+        else:
+            raise ArgumentIndexError(self, argindex)
+
+    @classmethod
+    def eval(cls, arg):
+        if arg.is_Number:
+            if arg is S.Zero:
+                return S.One
+            elif arg in [S.Infinity, -S.Infinity]:
+                return S.Zero
+            elif arg is S.NaN:
+                return S.NaN
+
+        if arg is S.ComplexInfinity:
+            return S.ComplexInfinity
```

Comment:
Yes, ``sinc`` has no limit at ``zoo`` ([an essential singularity](http://en.wikipedia.org/wiki/Essential_singularity)). I believe the value should be ``nan`` in SymPy.